src/transit/tests.py

```diff
@@ -6,6 +6,21 @@
 BASE_DEV_URL = environ["TRANSIT_DEV_BACKEND_URL"]
 BASE_PROD_URL = environ["TRANSIT_BACKEND_URL"]
 
+# We want to verify that /allstops returns a list of type 'busStop' only
+def allstops_returns_busStops(r):
+    response = r.json()
+    # Make sure response was successful
+    if not response["success"]:
+        return False
+
+    # Iterate over stops
+    for bus_stop in response["data"]:
```

Comment:
i think it makes more sense to call this `stop` so that when you actually verify it's a `busStop` it's not confusing that a `bus_stop` wouldn't be a `busStop`